pyanaconda/ui/gui/spokes/custom_storage.glade

```diff
@@ -1512,11 +1512,12 @@
                     </child>
                     <child>
                       <object class="GtkButton" id="resetButton">
-                        <property name="label" translatable="yes" context="GUI|Custom Partitioning">_Reset All</property>
+                        <property name="label" translatable="yes" context="GUI|Custom Partitioning">_Cancel All Changes</property>
                         <property name="visible">True</property>
                         <property name="sensitive">False</property>
                         <property name="can_focus">True</property>
                         <property name="receives_default">True</property>
+                        <property name="tooltip_text" translatable="yes" context="GUI|Custom Partitioning">Return to the state before any changes.</property>
```

Comment:
I wonder if "Roll back any changes" would be better ?

Reply:
Definitely!